aggregate-snapshot/aggregate-snapshot-repository/src/main/java/uk/gov/justice/services/eventsourcing/jdbc/snapshot/jdbc/snapshot/SnapshotJdbcRepository.java

```diff
@@ -106,24 +106,24 @@ protected String jndiName() throws NamingException {
         return String.format(JNDI_DS_EVENT_STORE_PATTERN, warFileName());
     }
 
+    @Override
+    protected AggregateSnapshot entityFrom(final ResultSet resultSet) throws SQLException {
+        return new AggregateSnapshot(
+                (UUID) resultSet.getObject(COL_STREAM_ID),
```

Comment:
Do we normally use getObject and a cast of do we use UUID.fromString and getString()?